src/app/dashboard/home/home.component.ts

```diff
@@ -255,10 +259,15 @@ export class HomeComponent implements OnInit, OnDestroy, AfterViewChecked, After
 
 
   changePicturePhotos(base64: string): void {
-    this.rucherService.rucher.photo = base64;
-    this.rucherService.updateBackgroundApiary(this.rucherService.rucher._id);
-    this.selectPhotoApiary = null;
+    if (base64.length * (3/4) < this.maxUploadFileSize) {
+      this.rucherService.rucher.photo = base64;
+      this.rucherService.updateBackgroundApiary(this.rucherService.rucher._id);
+      this.selectPhotoApiary = null;
+    } else {
+      this.notifyService.notify('warning', 'Le fichier est trop volumineux pour être téléchargé. (La taille maximale du fichier est de 6 Mo)');
+    }
```

Comment:
Good for the message,
one question : How are you managing this translation in EN for example?

Reply:
Just draft message in EN and translate it to  french  via google translator. I think, it is correct.

Reply:
But I mean that depending on user prefs for lang message should display in EN, FR or ES

Reply:
you have translations in src/assets/i18n/en.json fr.json and es.json